src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration.Tests/Azure.Configuration.Tests.csproj

```diff
@@ -9,20 +9,13 @@
     <PackageReference Include="System.Memory" Version="4.5.1" />
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="..\Azure.ApplicationModel.Configuration\Azure.Configuration.csproj" />
+    <ProjectReference Include="$(MSBuildThisFileDirectory)..\Azure.ApplicationModel.Configuration\Azure.Configuration.csproj" />
```

Comment:
Any particular reason you added the `$(MSBuildThisFileDirectory)`? While it doesn't hurt anything relative paths also resolve correctly so was curious why prompted you to use it.

Reply:
Nothing in particular, just that I've been hurt in the past with relative paths so I wanted to be explicit about them